homeassistant/components/media_player/cast.py

```diff
@@ -10,6 +10,7 @@
 from typing import Optional, Tuple
 
 import attr
+import pychromecast
```

Comment:
Why is this additional import needed? The import are already done where they are needed.

Reply:
Did you read the commit message?  The type checking on line 328 needs access the the type that is defined in the file that is proposed to be imported.

Reply:
This cannot be imported at the module scope because the import will not have been downloaded when the module is first loaded. The module is loaded and then the contents of the `REQUIREMENTS` constant are installed and then it will be importable.  See the dependencies section on the docs: https://developers.home-assistant.io/docs/en/development_checklist.html

Reply:
Thanks for the pointer @rohankapoorcom ...  Let's see if this works better.